tests/settings.py

```diff
@@ -161,3 +161,13 @@
     from local_settings import *
 except ImportError:
     pass
+
+REST_AUTH_SERIALIZERS = {
+    'PASSWORD_RESET_SERIALIZER': 'openwisp_radius.api.serializers.PasswordResetSerializer',
+}
+
+PASSWORD_RESET_URLS = {
+    'default': 'https://example.com',  # fallback in case the specific org page is not defined
```

Comment:
The entire password reset URL of the frontend must go here, someone may want to use a custom application which has a URL that is different than the openwisp captive page app will implement.

you should use:

```python
'https://example.com/{organization}/password/reset/{id}/{token}'
```